mobile/src/profile/styles.ts

```diff
@@ -87,30 +90,31 @@ export const contentStyles = StyleSheet.create({
     position: {
         fontSize: 13,
         textAlign: 'center',
-        color: '#000'
+        color: '#000',
+        top: 10
     },
     department: {
         fontSize: 13,
         textAlign: 'center',
         color: '#2FAFCC',
-        fontWeight: 'bold'
+        fontWeight: 'bold',
+        top: 15
     },
     infoContainer: {
         flexDirection: 'row',
-        padding: 5
+        padding: paddingInfoContainer,
+        top: 30
     },
     contactsContainer: {
-        paddingTop: 20,
+        top: 40,
         padding: 5,
-        alignItems: 'center',
-        justifyContent: 'center'
     }
 });
 
 export const tileStyles = StyleSheet.create({
     container: {
-        padding: 1,
-        width: '25%'
+        padding: paddingTile,
+        width: (Dimensions.get('window').width - 2 * paddingInfoContainer - 3 * paddingTile) / 4,
```

Comment:
it's really complicated.

What if instead of ```padding: paddingTile``` you try to add separate view with ```width: paddingTile``` between your tiles:

```javascript
        const TitleSeparator = () => <View style={{flexBasis: paddingTile /*initial size (width)*/, backgroundColor: '#fff'}}></View>;
...

        const lastIndex = tilesData.length - 1;

        return tilesData.map((tile, index) => (
            <React.Fragment>
                <View key={tile.label} style={tileStyles.container}>
                    <View style={tileStyles.tile}>
                        <View style={tileStyles.iconContainer}>
                            <ApplicationIcon name={tile.icon} size={tile.size} style={tile.style} />
                        </View>
                        <StyledText style={tileStyles.text}>{tile.label}</StyledText>
                    </View>
                </View>
                {
                    lastIndex !== index 
                        ? <TitleSeparator key={`${tile.label}-${index}`} />
                        : null
                }
            </React.Fragment>
        ));
```

Reply:
i will try